app/controllers/DayAggregateEventController.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import org.joda.time.LocalDate
+import play.api.libs.json.{ Json, OFormat }
+import play.api.mvc.{ Action, AnyContent }
+import repositories.events.EventsRepository
+import uk.gov.hmrc.play.microservice.controller.BaseController
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object DayAggregateEventController extends DayAggregateEventController {
+
+  override val eventsRepository: EventsRepository = repositories.eventsRepository
+
+}
+
+trait DayAggregateEventController extends BaseController {
+  def eventsRepository: EventsRepository
+
+  def findBySkills(skills: String): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
I would prefer you got one step further and implement a QueryStringBindable so that we could have define the controller action as ```findBySkills(skillTypes: List[SkillType]```. In any case not that strictly speaking the parameter(s) are SkillType not Skill.